adafruit_led_animation/animation/sparkle.py

```diff
@@ -57,18 +57,22 @@ class Sparkle(Animation):
     :param float speed: Animation speed in seconds, e.g. ``0.1``.
     :param color: Animation color in ``(r, g, b)`` tuple, or ``0x000000`` hex format.
     :param num_sparkles: Number of sparkles to generate per animation cycle.
+    :param mask: array to limit sparkles within range of the mask
     """
 
     # pylint: disable=too-many-arguments
-    def __init__(self, pixel_object, speed, color, num_sparkles=1, name=None):
+    def __init__(self, pixel_object, speed, color, num_sparkles=1, name=None, mask=[]):
         if len(pixel_object) < 2:
             raise ValueError("Sparkle needs at least 2 pixels")
+        if len(mask) >= len(pixel_object):
```

Comment:
if we go with `None` as the default parameter, then we can move this length check down below the next thing.